ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/MeteredLedgerDao.scala

```diff
@@ -52,6 +52,8 @@ class MeteredLedgerReadDao(ledgerDao: LedgerReadDao, metrics: MetricRegistry)
       metrics.timer("daml.index.db.remove_expired_deduplication_data")
   }
 
+  override def maxConcurrentConnections: Int = ledgerDao.maxConcurrentConnections
```

Comment:
```suggestion
  override val maxConcurrentConnections: Int = ledgerDao.maxConcurrentConnections
```
Is it safe to turn this into a `val`?

Reply:
Well, the trait has it as a `def`, so theoretically, no, but in practice, yes.

Reply:
You can always override a `trait`s `def` with a `val`. :wink:

Reply:
I think I'd like to leave it as a `def` for now.

BTW, this is only used on application startup, once, by `SqlLedger`, which is going to be deleted in the next couple of months, so this should disappear too.

Reply:
That's ok, it's not a problem, but can you clarify me why you'd like to leave it as a `def`?